exonum-java-binding/core/rust/integration_tests/src/vm.rs

```diff
@@ -103,26 +105,45 @@ fn get_fakes_classpath_option() -> String {
 }
 
 pub fn get_fakes_classpath() -> String {
-    let classpath_txt_path = project_root_dir().join("fakes/target/ejb-fakes-classpath.txt");
+    let classpath_txt_path =
+        java_binding_parent_root_dir().join("fakes/target/ejb-fakes-classpath.txt");
 
     let mut class_path = String::new();
     File::open(classpath_txt_path)
         .expect("Can't open classpath.txt")
         .read_to_string(&mut class_path)
         .expect("Failed to read classpath.txt");
 
-    let fakes_path = project_root_dir().join("fakes/target/classes/");
+    let fakes_path = java_binding_parent_root_dir().join("fakes/target/classes/");
     let fakes_classes = fakes_path.to_str().expect(CONVERSION_FAILED_MESSAGE);
 
     // should be used `;` as path separator on Windows [https://jira.bf.local/browse/ECR-587]
     format!("{}:{}", class_path, fakes_classes)
 }
 
+/// Returns a Log4j system property pointing to the configuration file. The file is in
+/// the base directory of the `integration_tests` project and must be edited if more detailed
+/// Java logs are needed for debugging purposes.
+///
+/// It requires the log4j-core library to be present on the classpath, which is the case with fakes.
+fn get_log4j_path_option() -> String {
```

Comment:
nit: why we even use these `get_` prefixes? https://rust-lang-nursery.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter